src/app/shared/models/classes/coffee.class.ts

```diff
@@ -25,24 +25,24 @@ export class Coffee extends Product {
             }
 
             if (coffeeShape.tastingNotes != null) {
-                this.tastingNotes = coffeeShape.tastingNotes;
+                this.tastingNotes = (coffeeShape.tastingNotes as any as string).split(", ");
```

Comment:
The back-end still passes the `coffeeShape` with tasting notes as a single string, so I just parsed it manually here. Obviously, when the back-end changes we'll need to update this.